INDEX

```diff
@@ -1,4 +1,12 @@
 {
+  "bazarr": {
+    "MANIFEST": "bazarr.json",
+    "description": "Bazarr is a companion application to Sonarr and Radarr that manages and downloads subtitles.",
+    "icon": "https://icons.freenas.org/community-icons/bazarr.png",
+    "name": "Bazarr",
+    "official": false,
+    "primary_pkg": ""
```

Comment:
If there is no suitable primary pkg, enter "null".

Reply:
Ah sure thing. Fixed